_connect-files/api/objects/form-properties/sources/saas/darksky-object.md

```diff
@@ -0,0 +1,71 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/connect-templates/destination-form-property/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#        CONTENT TYPE        #
+# -------------------------- #
+
+content-type: "api-form"
+form-type: "source"
+key: "source-form-properties-darksky-object"
+
+
+# -------------------------- #
+#        OBJECT INFO         #
+# -------------------------- #
+
+title: "Darksky Source Form Property"
+api-type: "platform.darksky"
+display-name: "Darksky"
+
+source-type: "saas"
+docs-name: "darksky"
+
+property-description: ""
+## Used to create a description for the object that doesn't adhere to the standard in _developers/connect/api/documentation/api-form-properties.html
+## See the Heap object for an example
+
+
+# -------------------------- #
+#      OBJECT ATTRIBUTES     #
+# -------------------------- #
+
+uses-start-date: true
+
+# Only source-specific attributes need to be listed here.
+# The following attributes are considered common,
+# and therefore don't need to be listed:
+# anchor_time, cron_expression, frequency_in_minutes, image_version, start_date 
+
+object-attributes:
+  - name: "language"
+    type: "string"
+    required: true
+    description: "The language."
```

Comment:
Based on Darksky's API docs, this is what this setting does:

> Return `summary` properties in the desired language.

Meaning that the `summary` attributes in the `forecast` table are returned in this language. I'd make this description more specific, and also include a link to the list of values that the API will accept.

I see that their API docs don't include heading anchors, so maybe something like this:

```suggestion
    description: |
      The language that `summary` attributes in the `forecast` table should be returned in. For example: If set to `en`, `summary` data will be returned in English.
      
      Refer to [{{ form-property.display-name }}'s documentation](https://darksky.net/dev/docs#time-machine-request){:target="new"} for a list of accepted values.
```